.github/workflows/publish-to-ghcr.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Docker Images to GHCR
+
+on:
+  push:
+    branches:
```

Comment:
We should also trigger for `release/*` branches

Reply:
Ok, what we want ? 

- Trigger on tags ?
- Trigger on master ?
- Trigger on release/* ?

Reply:
I think that we should trigger on pushes to master and release/* and tag with the commit SHA for the time being.